.github/ISSUE_TEMPLATE/package.yml

```
@@ -0,0 +1,60 @@
+name: Packages
+description: Report problems and suggest ideas for the packages
+labels: [package]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `package` label][package-issues] (including
+        closed issues) before you continue.
+
+
+        [package-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Apackage+is%3Aopen
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: >-
+        If your issue type is not here, select "other" and explain in the
+        "Description" field below.
+      options: [improvement, bug, other]
```

Minor, should we capitalize these?

```suggestion
      options: [Improvement, Bug, Other]
```